modules/sdk/gradle-plugins-defaults/CHANGELOG.markdown

```diff
@@ -419,6 +419,12 @@ the `com.liferay.app.tlddoc.builder` plugin.
 - [LRDOCS-3023]: Update the [Liferay Gradle Plugins] dependency to version
 3.0.4.
 
+## 1.2.49 - 2016-10-09
+
+### Added
+- [LRDOCS-2594]: apply Liferay stylesheet file, if found, to the API
+documentation generated by the `appJavadoc` and `javadoc` tasks.
```

Comment:
@Ithildir which Liferay stylesheet file are we talking about here? Do you think it would be helpful to provide that file name?